buildroot/arm.config

```diff
@@ -1,5 +1,8 @@
 BR2_arm=y
 BR2_OPTIMIZE_3=y
+BR2_KERNEL_HEADERS_4_4=y
+BR2_BINUTILS_VERSION_2_31_X=y
+BR2_GCC_VERSION_8_X=y
```

Comment:
github shows the binary size decreased by 99%, that can't be right, can it?

Reply:
I think it means that 99% of it was rewritten but maybe not... I have zero idea.